### ISSUE TITLE
[Suggestion] Mention of repo is deprecated

### ISSUE THREAD

**Issue by hmenzagh** [reHackable/awesome-reMarkable#93]
Mention in the lists if the repo is deprecated or not.
Something like : 

- 🔴 CUPS Printing - Script to print directly to reMarkable Cloud from CUPS using rMAPI.
- ✅ reMarkable_syncthing - Syncthing on reMarkable.

**Comment by Barabazs** [comment]
As these are all individual third party repos/scripts/... they will never be "deprecated" in the programming sense.
Wouldn't it be better to tag a repo as "abandoned" instead? But keep in mind that an abandoned repo could still be revived later on.
Who will keep track of this status? :man_shrugging:

**Comment by Ameb** [comment]
I agree with Barabazs. Even the awesome list have been abandoned for a while (till @Barabazs and @Liblor stepped in).

**Comment by axkibe** [comment]
> As these are all individual third party repos/scripts/... they will never be "deprecated" in the programming sense. Wouldn't it be better to tag a repo as "abandoned" instead? But keep in mind that an abandoned repo could still be revived later on. Who will keep track of this status? man_shrugging

Well since the cloud API has recently changed a bunch of project actually are "deprecated" in a sense they don't work anymore. That would be super helpful in the list to kind of mark the projects that just won't work. (they might again, if someone picks it up and adapts it to the new API.. which is as far I know realized currently still not known)

**Comment by Eeems** [comment]
I've started marking archived repos with `(Unmaintained)` for now. #177 may eventually have a better solution for keeping track of what is supported by specific repos.
